docs/toc.md

```diff
@@ -133,7 +133,7 @@
 ## [Installation guide](framework/install/)
 ## [Migration Guide](framework/migration-guide/)
 ## [.NET Framework on Docker Guide](framework/docker/index.md)
-### [Running Console Apps in Containers](framework//docker/console.md)
+### [Running Console Apps in Containers](framework/docker/console.md)
 ## [Development Guide](framework/development-guide.md)
```

Comment:
the development guide TOC is both listed here and on the sub-TOC. I'd either keep the entries on the main TOC or on the sub-TOC, not both. (see https://review.docs.microsoft.com/en-us/dotnet/framework/development-guide?branch=pr-en-us-2399)